contracts/transmuter/facets/SettersGovernor.sol

```diff
@@ -90,6 +91,11 @@ contract SettersGovernor is AccessControlModifiers, ISettersGovernor {
         LibSetters.setOracle(collateral, oracleConfig);
     }
 
+    function updateOracle(address collateral) external {
```

Comment:
Why is this in "SettersGovernor"?

Reply:
It'd be cleaner to have it in RewardHandler no? With the same access control as `sellRewards`

Reply:
Overall we could also change the naming of `isSellerTrusted` and the associated getters to something like `isTrustedKeeper` and have a clean comment somewhere expliciting the rights these addresses have